### DISCUSSION TITLE
Feature request: Provide a shortcut to open link in split view

### DISCUSSION THREAD

**Discussion by aohoyd** [zen-browser/desktop#2030] [Ideas]
I would like to have a shortcut (ctrl+click e.g.) to open a link in a Split View

**Comment by spacefren** [comment]
Bump, it would be great to have a shortcut for the "split link to new tab" option in the link context menu.

`ctrl + click` is already taken, so maybe `alt + click` could work?

**Comment by Dedsecproject** [comment]
They just added that or it was under an unfamiliar name. You can open the link by right-clicking on it, then selecting `split link to a new tab,` and presto!
![SCR-20241123-novg](https://github.com/user-attachments/assets/2c8fb8b3-7f94-480f-b8ef-90d31769f4c5)

**Reply by laurentthiebaudezm** [reply]
I agree with the original question. That would be nice to have a shortcut, in order to avoid the right click.

**Comment by JordanTranchina** [comment]
Vibecoded this feature here, if anyone is able to test it. https://github.com/zen-browser/desktop/pull/9721

**Comment by yokhoe** [comment]
have the devs responded to this yet? whether it's in a roadmap?
